test/src/net.rs

```diff
@@ -51,7 +58,7 @@ impl Net {
             controller: None,
             p2p_port,
             setup,
-            working_dir: temp_path(case_name, "net"),
+            working_dir: case_working_dir.join("net").as_path().display().to_string(),
```

Comment:
unnecessary as_path

```
working_dir: case_working_dir.join("net").display().to_string(),
```

Reply:
It would be a good chance for us to refactor working_dir field to `PathBuf` in this PR.